libyui-ncurses/pkgconfig/libyui-ncurses.pc.in

```diff
@@ -0,0 +1,23 @@
+#
+# Pkg-Config file for libyui-ncurses
+#
+# Generated by CMake from libyui-ncurses.pc.in
+#
+
+prefix=@CMAKE_INSTALL_PREFIX@
+exec_prefix=${prefix}
+
+libdir=${exec_prefix}/@CMAKE_INSTALL_LIBDIR@
+includedir=${exec_prefix}/include/yui/ncurses
+plugindir=${exec_prefix}/@CMAKE_INSTALL_LIBDIR@/yui
```

Comment:
I'm not sure plugindir is needed outside libyui. Plugins and extensions too should (most probably must) use the one provided from libyui.pc

Reply:
It's used further down in this file as you found out below... ;-)